### DISCUSSION TITLE
GitHub Student Developer Pack Verification Email Not Received

### DISCUSSION THREAD

**Discussion by MuhammadShoaibKalim** [community/community#147903] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

Hello GitHub Community,

I've been a GitHub user from 2021 to now, but I haven’t received the email to verify my account to get my developer student pack benefits. I have checked my spam/junk folder and waited for more than 3 days, but still no email has arrived. Could you please assist me with this issue?

I would appreciate any guidance or help in getting my account verified.

Thank you!
Shoaib

**Comment by mecodeatlas** [comment]
Thanks for posting in the GitHub Community, @MuhammadShoaibKalim!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the General category. I've gone ahead and moved it for you. Good luck!
